backend/internal/kyc_client.go

```diff
@@ -160,3 +161,35 @@ func (c *KYCClient) IsUserVerified(ctx context.Context, sponseeAddress string) (
 
 	return result.Result.Status == "VERIFIED", nil
 }
+
+// IsValidSponsee checks if a sponsor is a valid sponsee and not white listed by calling the tf-kyc-verifier API
+func (c *KYCClient) IsValidSponsee(ctx context.Context, sponsorAddress string) (bool, error) {
+	if strings.TrimSpace(sponsorAddress) == "" {
```

Comment:
This isn't a strong enough validation

Reply:
@0oM4R
If you need proper SS58 validation, Check https://github.com/vedhavyas/go-subkey/blob/adc49f77cad829c4b6d541c46c9e3f44e2afcc26/ss58_address.go#L12